Client/Frontend/Browser/Browser.swift

```diff
@@ -11,8 +11,14 @@ protocol BrowserHelper {
     func userContentController(userContentController: WKUserContentController, didReceiveScriptMessage message: WKScriptMessage)
 }
 
+protocol BrowserDelegate {
+    func snackbarAdded(bar: SnackBar)
```

Comment:
Lol. Its an Android widget. We could use something like NotificationBar or InfoBar?